test/unit/qrda/patient_round_trip_test.rb

```diff
@@ -55,6 +68,32 @@ def generate_shell_patient(type)
         cqm_patient
       end
 
+      def test_interval_round_trips
```

Comment:
This method was a part of commit "Allow for low and high times to be nullFlavored" and not a part of the patient addresses branch. Do we want to include it here?